uint8           motor_num
---
bool            success
TmcParam[]      param